src/wallet/index.ts

```diff
@@ -86,6 +86,25 @@ export class Wallet {
     }
   }
 
+  static verifySignature = async (signature: Signature, tx: Transaction): Promise<boolean> => {
+    const serializedTx = await tx.serialize()
```

Comment:
Why is it async ?

Reply:
Because we are handling CJS and ESM support on some modules that are ESM only.